wiki/templates/wiki/move.html

```diff
@@ -0,0 +1,75 @@
+{% extends "wiki/article.html" %}
+{% load wiki_tags i18n sekizai_tags %}
+
+{% block wiki_pagetitle %}{% trans "Move" %}: {{ article.current_revision.title }}{% endblock %}
+
+{% block wiki_contents_tab %}
+
+{% if not user.is_authenticated %}
```

Comment:
Is this page eligible for anonymous users? Was just wondering if it makes sense or if there's a danger or complexity in allowing that?